product_unique_serial_picking/model/product.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+from openerp import fields, models
+
```

Comment:
This file is duplicated with:
https://github.com/Vauxoo/stock-logistics-workflow/blob/8.0/product_unique_serial/model/product.py